packages/time-picker/src/time-picker-com/panel-time-range.vue

```diff
@@ -125,6 +126,13 @@ export default defineComponent({
     const minSelectableRange = ref([])
     const maxSelectableRange = ref([])
 
+    // watch
+    watch(() => props.visible, val => {
+      if (!val) {
+        oldValue.value = props.parsedValue
+      }
+    })
+
```

Comment:
Since the code is sort of the same here, can we encapsulate them together with some `useXxxx` hook to avoid duplication?

Reply:
Yes, it is better.

Reply:
done~